src/AppBundle/Issues/StatusApi.php

```diff
@@ -45,6 +45,16 @@ public function addIssueLabel($issueNumber, $newLabel)
         return true;
     }
 
+    /**
+     * @param int $issueNumber The GitHub issue number
+     *
+     * @return string[]
+     */
+    public function getLabels($issueNumber)
+    {
+        return $this->labelsApi->getIssueLabels($issueNumber);
```

Comment:
Could it be in a property collection?
In case there is many getLabels() calls

Reply:
`$this->labelsApi` is an instance of `CachedLabelsApi` that already keeps fetched labels in cache

Reply:
\o/